src/bit-components.js

```diff
@@ -197,3 +197,17 @@ export const ObjectSpawner = defineComponent({
 export const Billboard = defineComponent({
   onlyY: Types.ui8
 });
+export const MaterialTag = defineComponent();
+export const UVScroll = defineComponent({
+  speed: [Types.f32, 2],
+  increment: [Types.f32, 2],
+  offset: [Types.f32, 2]
+});
+export const VideoTextureSource = defineComponent({
+  fps: Types.ui8,
+  resolution: [Types.ui16, 2]
+});
+export const VideoTextureTarget = defineComponent({
+  srcNode: Types.eid,
```

Comment:
`srcNode` -> `source`? (May not be worth the trouble since gltf components say `srcNode` now.)